### PR TITLE
initial setup of repo

### PR DESCRIPTION
**Pull request by scottford-lw**
This PR sets up the initial files for development of the module. 

Signed-off-by: Scott Ford <scott.ford@lacework.net>

### CODE REVIEW

File: scripts/ci_tests.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# Description:: Use this script to run ci tests of this repository
+# Author::      Salim Afiune Maya (<afiune@lacework.net>)
+#
+set -eou pipefail
+
+readonly project_name=terraform-kubernetes-agent
+
+TEST_CASES=(
+  examples/custom-resource-allocation
+  examples/custom-tolerations
```
Comment (afiune) [inline scripts/ci_tests.sh:13]: these tests don't exist.
```suggestion
```

File: scripts/ci_tests.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  examples/default
```
Comment (afiune) [inline scripts/ci_tests.sh:14]: this test does exist but is empty. Not sure if we want to add it.

File: scripts/ci_tests.sh [hunk 3]
```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+#
+# Name::        ci_tests.sh
+# Description:: Use this script to run ci tests of this repository
+# Author::      Salim Afiune Maya (<afiune@lacework.net>)
+#
+set -eou pipefail
+
+readonly project_name=terraform-kubernetes-agent
```
Comment (afiune) [inline scripts/ci_tests.sh:9]: copy/paste 😉
```suggestion
readonly project_name=terraform-aws-ssm-agent
```

File: scripts/release.sh [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+# Name::        release.sh
+# Description:: Use this script to prepare a new release on Github,
+#               the automation will create a GH tag like 'v0.1.0'
+#               (using the VERSION file)
+# Author::      Salim Afiune Maya (<afiune@lacework.net>)
+#
+set -eou pipefail
+
+readonly org_name=lacework
+readonly project_name=terraform-kubernetes-agent
```
Comment (afiune) [inline scripts/release.sh:12]: this is really important:
```suggestion
readonly project_name=terraform-aws-ssm-agent
```

File: scripts/release.sh [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+readonly git_user="Alan Nix"
+readonly git_email="alan.nix@lacework.net"
```
Comment (afiune) [inline scripts/release.sh:14]: Let us start using our Lacework github account for these commits:
```suggestion
readonly git_user="Lacework Inc."
readonly git_email="ops+releng@lacework.net"
```

**Review by afiune — CHANGES_REQUESTED** [review]
A number of changes needed plus the `VERSION` file that is missing.

File: scripts/ci_tests.sh (hunk 2 above)
Comment (scottford-lw) [reply scripts/ci_tests.sh:12]: Sounds good @afiune!

**Review by afiune — APPROVED** [review]
tenor-25492744

**Pull request merged**